src/components/Publish.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="publish-wrapper">
+    <h2>Publish</h2>
+    <form class="publish-form" method="POST" action="/" enctype="multipart/form-data">
+      <div>
+        <input type="file" name="file" />
+      </div>
+      <div>
+        lbry://<input type="text" name="name" />
+      </div>
+      <input type="submit" name="publish" value="Go" />
+      <p class="help-text">Publishing can take a few moments. Please be patient.</p>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  data: () => {
+    return {
+      showClaimInput: false
+    }
+  }
+}
+</script>
+
+<style>
+  .publish-wrapper{
```

Comment:
wasn't quite sure which spacing you preferred, but I will update to match the style tag in index.vue